### PR TITLE
Added to Genesis and Consensus mechanism sections

### PR DESCRIPTION
**Pull request by captainbuckkets**
Expanded on sections.  Used Peercoin University for reference.

### CODE REVIEW

File: app/assets/docs/002-comparison-with-other-blockchains.md [hunk 1]
```diff
@@ -5,7 +5,9 @@ Major differences between the two are the fee market, ie. the absence of it in P
 
 ## Consensus algorithm
 
-Peercoin is secured by Proof-of-stake type of consensus.
+Peercoin is secured by Proof-of-stake type of consensus.  Proof-of-work uses energy as a scary resource to verify transactions, while proof-of-stake uses time as a scare resource measured in "coin age".  Once a transaction has reached an age of thirty (30) days, these coins become eligible for minting.  With a high enough coin, a new block will be minted, with minting being determined by fixed probability functions.  After minting, a portion of will be given back to the minter, and their coins will again begin the thirty (30) day maturation process, giving other minters the opportunity to mint blocks.  As a note, after ninety (90) days, a transaction reaches its maximum maturity and minting probability is at its highest.  This minting can be done by simple low energy computers, such as a raspberry pi, making the process energy efficient when compared to proof-of-work power consumption, further increasing the possibility of decentralization.
```
Comment (peerchemist) [inline app/assets/docs/002-comparison-with-other-blockchains.md:8]: ...as a *scarce* resource

File: app/assets/docs/001-introduction.md [hunk 2]
```diff
@@ -4,7 +4,9 @@ Hello, and welcome to the Peercoin Documentation website. We hope to help you un
...
+Since the creation of Bitcoin (Nakamoto 2008), proof-of-work has been the predominant design of peer-to-peer crypto currency. The concept of proof-of-work has been the backbone of mining and security model of Nakamoto’s design.  In 2012, Sunny King and Scott Nadal proposed an alternative form of concensus that was both more secure, but also energy efficient.  Proof-of-work, however is subject to centralized mining, large amounts of power consumption, and majority attacks.  Proof-of-stake sought to provide sustainability and improved security through energy efficient staking and time-based confirmations (coin age).  Instead of electricity and computing power, time could be used as a verification method that would prevent many of the issues plaguing proof-of-work consensus.  Proof-of-stake guards the blockchain, keeping users safe, while dynamic proof-of-work mining provides economic competition and maintains a balance in distribution.  Dynamic mining and staking allow for around 1% inflation a year, making it extremely practical for long term use.
```
Comment (peerchemist) [inline app/assets/docs/001-introduction.md]: > Proof-of-work, however is subject to centralized mining,

Too generic. I'd say it's susceptible to miner centralization as consequence of natural process of economies of scale.

**Comment by peerchemist** [comment]
Left some comments on this in case you have not noticed.

**Comment by captainbuckkets** [comment]
I might have double sent.  Still learning Git I guess...

**Pull request merged**